src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -285,10 +293,16 @@ private String getPendingOrders() throws Exception {
 				throw new FileNotFoundException(e.toString());
 			}
 		} catch (FileNotFoundException e) {
+			try {
+				in = new BufferedReader(new InputStreamReader(baseURL2.openStream()));
+			} catch (IOException t) {
+				e.getMessage();

```

Comment:
This returns a string, we're doing nothing with it here. Also we're getting the parent exception `e` and not the current exception `t`.